server/models/articles.model.js

```diff
@@ -12,8 +20,17 @@ exports.fetchArticle = (article_id) => {
     })
 }
 
-exports.fetchArticles = () => {
-    return db.query(`
+exports.fetchArticles = (topic = '%') => {
+    const topicQuery = format(`
+    SELECT * FROM topics
+    WHERE slug LIKE %L;
+    `,topic)
+    return db.query(topicQuery) //refactor this to be its own function?
```

Comment:
Please remove comments before committing. Although I agree this being a utils function that we call alongside this model in the controller with a promise.all could be a nice way of handling it 👍